cscs-checks/cuda/multi_gpu.py

```diff
@@ -41,15 +44,21 @@ def __init__(self):
         if self.current_system.name in ['daint', 'dom', 'tiger']:
             self.modules = ['craype-accel-nvidia60']
             self.num_gpus_per_node = 1
-        else:
-            self.modules = ['craype-accel-nvidia35']
+        elif self.current_system.name == 'kesch':
+            self.modules = ['cudatoolkit/8.0.61']
+            self.num_gpus_per_node = 8
+        elif self.current_system.name in ['arolla', 'tsa']:
+            self.modules = ['cuda/10.1.243']
             self.num_gpus_per_node = 8
 
         # perf_patterns and reference will be set by the sanity check function
         self.sanity_patterns = self.do_sanity_check()
         self.perf_patterns = {}
         self.reference = {}
         self.__bwref = {
+            'arolla:cn:h2d':   (7583, -0.1, None, 'MB/s'),
+            'arolla:cn:d2h':   (7584, -0.1, None, 'MB/s'),
+            'arolla:cn:d2d': (137408, -0.1, None, 'MB/s'),
```

Comment:
I am not sure about these numbers. Let's discuss them.

Reply:
I copied all reference values from Kesch: they can be adapted at a later stage. 
Only if a check was failing I have adapted them immediately to the values on the new system.

Reply:
See https://jira.cscs.ch/browse/UES-714